anaconda-mode.el

```diff
@@ -632,10 +632,15 @@ PRESENTER is the function used to format buffer content."
   "Find DEFINITION file other window, go to DEFINITION point."
   (anaconda-mode-find-file-generic definition 'find-file-other-window))
 
-(defvar-local anaconda-mode-go-back-definition nil
+(defun anaconda-mode-find-file-no-record-definition (definition)
+  "Find DEFINITION file, go to DEFINITION point (without recording in the go-back stack)"
+  (anaconda-mode-find-file-generic definition 'find-file t)
+  )
```

Comment:
Don't add new line before `)`. Always keep closing brackets on there same line. This is common style for all languages in the Lisp family ;)